Tasks/JenkinsQueueJob/job.ts

```diff
@@ -98,7 +107,7 @@ export class Job {
             } else if (oldState == JobState.Streaming) {
                 validStateChange = (newState == JobState.Finishing);
             } else if (oldState == JobState.Finishing) {
-                validStateChange = (newState == JobState.Downloading || newState == JobState.Done);
+                validStateChange = (newState == JobState.Downloading || newState == JobState.Queued || newState == JobState.Done);
```

Comment:
Here "queued" does not mean queued to be run.  Queued means that this task queued the job on Jenkins and is now done because the user disabled the "Capture console output..." setting.  In other words, the task will queue the job but not wait for it to complete, so the task considers it done.  See the `finish()` function for this logic.